libraries/AP_Notify/AP_Notify.cpp

```diff
@@ -228,9 +229,11 @@ void AP_Notify::add_backends(void)
                 break;
             case Notify_LED_ToshibaLED_I2C_Internal:
                 ADD_BACKEND(new ToshibaLED_I2C(TOSHIBA_LED_I2C_BUS_INTERNAL));
+                ADD_BACKEND(new NCP5623(TOSHIBA_LED_I2C_BUS_INTERNAL));
```

Comment:
Add a new enum value for Notify_LED_NCP5623, and set the bit in the define appropriately for any boards it should be auto detected on. You should not be overloading the ToshibaLED to also create the NCP driver.